metadata.json

```diff
@@ -38,30 +38,26 @@
     {
       "operatingsystem": "Scientific",
       "operatingsystemrelease": [
-        "5",
         "6",
         "7"
       ]
     },
     {
       "operatingsystem": "SLES",
       "operatingsystemrelease": [
-        "11 SP1",
         "12"
       ]
     },
     {
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
-        "7",
         "8",
         "9"
       ]
     },
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "14.04",
```

Comment:
how come ubuntu 14.04 being removed, its not EOL until April 2019[1]

[1]https://wiki.ubuntu.com/Releases

Reply:
Hi @b4ldr. Ubuntu 14.04 is no longer supported as a master platform by Puppet. We are going through a process for adding Ubuntu 18 support to modules that are supported on Ubuntu, and in preparation for that are removing support for Ubuntu 14.04. Our pipelines will continue to test against Ubuntu 16. We're not making any actual code changes, so it should still work on 14.04. Let me know if you've any concerns. Thanks.

Reply:
ok thanks for the explanation

Reply:
@b4ldr Perhaps of interest to you, we changed to keeping 14.04 based on Puppet *agent* platform support rather than Puppet master platform support (so may be more along the April 2019 timeline that you mentioned).